### DISCUSSION TITLE
Does Storybook have a Rich Text control?

### DISCUSSION THREAD

**Discussion by danspratling** [storybookjs/storybook#17887] [Ideas]
In most situations when working with a CMS your content will be presented as Rich Text

When trying to render this content in Storybook, it's presented as an array of objects like so (I'm not sure if this differs depending on the CMS or the type of wysiwyg editor)

![Screenshot 2022-04-05 at 13 07 30](https://user-images.githubusercontent.com/11349104/161750304-03f97c72-6cd8-49e3-8979-afa6ddc87882.png)

What I would love, is a markdown or wysiwyg editor here interprets the rich text array and allows the user to edit it in a more typical way - I think this is a feature request as I can't find anything referencing a rich text control.

**Comment by williamlarsson** [comment]
Totally!
Looking for this feature at the moment. 
Storybook is so versatile and can be used in so many different situations and variations. 
Adding a Wysiwyg shoulnd't be that much of a hassle

**Comment by chantastic** [comment]
At the moment there isn’t a way to do this.
But if you want to make the controls more user-friendly, there is a lesser used feature for mapping dropdown options to complex implementations.

Here’s a video on that at the correct timestamp: https://youtu.be/0gOfS6K0x0E?si=BXZXL_vSp_xcRxvK&t=225
